src/state.ts

```diff
@@ -59,26 +83,27 @@ export interface StateNetworkInterface {
 
 export interface StateWalletInterface {
   selected: boolean;
-  identity?: IdentityInterface;
+  addressesWithBlindingKey: AddressInterface[];
   keystore: {
     type: KeyStoreType;
     value: string;
   };
 }
 
 export function getWalletInfo(walletState: StateWalletInterface): string {
```

Comment:
I think this belongs more to helpers than here. I would accept a list of addressesWithBlindingKey directly and craft the string as needed. I would call `logWalletInfo` or something like that